lib/src/robot/client.dart

```diff
@@ -10,11 +13,19 @@ import '../resource/registry.dart';
 import '../rpc/dial.dart';
 import '../viam_sdk.dart';
 
+Logger _logger = Logger();
+
 /// The options that define the behavior of the [RobotClient].
 class RobotClientOptions {
   /// Options for connecting to the robot
   final DialOptions dialOptions;
 
+  /// The frequency (in seconds) at which to check if the robot is still connected. 0 (zero) signifies no connection checks
+  final checkConnectionInterval = 1;
```

Comment:
I believe in other sdks the default is 10s

Reply:
ooh you are correct, i changed this to 1 for debugging purposes. Good catch, thank you!